formula-requirements.txt

```diff
@@ -1,2 +1,14 @@
-git@github.com:ministryofjustice/repos-formula.git
-git@github.com:ministryofjustice/nginx-formula.git
+git@github.com:ministryofjustice/nginx-formula.git==v3.2.1
+git@github.com:ministryofjustice/supervisor-formula.git==v1.1.0
```

Comment:
This is really out of my control as ```formula-requirements.txt``` is generated from ```metadata.yml``` by ```salt-shaker```. One of the formulas must be pulling it in as a dependency - my guess would be ```python-formula```.